### PR TITLE
Document which Fura Code variant to use for ligatures

### PR DESCRIPTION
**Pull request by sideeffffect**
#### Description

Fura Code doc improvment

related discussion in ticket https://github.com/ryanoasis/nerd-fonts/issues/248

### CODE REVIEW

**Comment by ryanoasis** [comment]
@sideeffffect Thank you and appreciate your change. I just read through #248 for context.

So this is the place that makes sense, however the readmes in the patched fonts are re-generated for each release so any changes to those files will be overwritten.

Your doc/readme change actually needs to go here: [/src/readme-per-directory-addendum.md](https://github.com/ryanoasis/nerd-fonts/blob/master/src/readme-per-directory-addendum.md) properly get propogated to all the readmes (probably better anyway that just Fura Code)..

I know that is not very clear and I am all for improving the documentation process that these readmes get generated.

Here is a basic overview :smiley: :
- [release script](https://github.com/ryanoasis/nerd-fonts/blob/master/bin/scripts/release.sh#L16) executes the standarized-and-complete readmes script
- the [standardize-and-complete-readmes script](https://github.com/ryanoasis/nerd-fonts/blob/master/bin/scripts/standardize-and-complete-readmes.sh#L105) appends the [readme-per-directory-addendum.md](https://github.com/ryanoasis/nerd-fonts/blob/master/src/readme-per-directory-addendum.md)

File: patched-fonts/FiraCode/readme.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ See the [Reserved Font Name section][SIL-RFN] for additional information
   * If you are limited to monospaced fonts (because of your terminal, etc) then pick a font with the `'Mono'` suffix.
     * This denotes that the Nerd Font glyphs will be monospaced not necessarily that the entire font will be monospaced
 
+### Ligatures
```
Comment (ryanoasis) [inline patched-fonts/FiraCode/readme.md]: Make the changes in the [readme-per-directory-addendum.md](https://github.com/ryanoasis/nerd-fonts/blob/master/src/readme-per-directory-addendum.md) instead :smiley:

Comment (sideeffffect) [reply patched-fonts/FiraCode/readme.md]: :heavy_check_mark:

**Comment by ryanoasis** [comment]
updating milestone and contribs

**Pull request merged**